### DISCUSSION TITLE
How to logout or revoke properly from application side?

### DISCUSSION THREAD

**Discussion by johnjhr** [etsy/open-api#1161] [Q&A] [answered]
Local web application configured with etsy api.
User logs in using oauth.
Listing can be created.
Now user needs to logout, clears every local session details like token, expiry, code etc, but the cookie/session information at etsy.com is still valid.
For example, If I open etsy.com in new tab, the user is still signed in. 
New sign in attempt is automatically logging in without asking for username and password until manually logged out from etsy.com.

current workaround is opening : www.etsy.com/logout.php in a new tab or clearing cookies that belong to etsy.

Is there a revoke or signout api path that can be used to revoke the current token?

**Comment by Gareth064** [comment, marked as answer]
There are no API's which touch Etsy User Login/Logout activities. You will have to continue with the workaround.
